presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.245
+=============
+
+**Highlights**
+==============
+* New `ENUM_KEY` UDF to get the key for an enum value.
+* Configurable query result compression.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix wrong results issue for queries with multiple lambda expressions that differ only in array/map/row constants that have the same length (:issue:`15424`).
+* Fix enum deserialization bug in map keys.
+* Improve memory usage for fragment result caching.
+* Add support for disabling query result compression via client and server-side configuration properties. Clients can disable compressed responses using the ``--disable-compression`` flag or ``disableCompression`` driver property. Compression can be disabled server-wide by using the configuration property: ``query-results.compression-enabled=false``
+* Add support in affinity scheduler to enable cache for bucketed table scan that has remote source.
+* Add a UDF to get the key corresponding to an enum value: `ENUM_KEY(EnumType) -> VARCHAR`
+* Add SqlParserOptions to prepared statement handler so that parsing logic is consistent with regular requests.
+
+Hive Connector Changes
+______________________
+* Improve parquet metadata reader by pre-loading 16KiB from the end of the file. When the metadata section is smaller than the prefetch size, only a single read is performed compared to the previous two.
```

Comment:
``Improve parquet metadata reader by preloading data and reducing the number of reads` - I don't think we should dig too much into the implementation details. We should just link to the issue/pr

Reply:
The release note generation logic doesn't link to PRs. I added the link for this one, but generally what are we doing? Seems like we should do it automatically.